### PR TITLE
CAPX-41 | Importer in use block + Orphan processing do nothing action

### PR DESCRIPTION
**Pull request by sherakama**
# Ready for review

To test:
1. Create a new importer complete with mapper
2. Import some profiles
3. Edit importer to see the block at the top of the page
4. Ensure that the number of profiles imported is correct
5. Click the configuration link and the view profiles button. View profiles should take you to a filtered view by machine name.

The orphan functionality that snuck into this PR is pretty simple. If the user selects the option to do nothing with orphaned profiles then don't bother checking for them at all as that is extra processing overhead.

### CODE REVIEW

**Comment by sherakama** [comment]
@pingers @mpriest,

Please review.

**Comment by RoSk0** [comment]
@sherakama 
Functionality review:
- The block didn't appeared on importer edit page.
- Number of profiles is correct.
- "View all imported profiles" link is leading to a view with correctly filled in importer in filter, but the view is empty.

File: includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php [hunk 1]
```diff
@@ -341,15 +347,22 @@ public function processOrphans($profileIds) {
 
       switch ($action) {
         case "delete":
-          drush_log("Deleted orphaned profile: " . $profile->label(), "status");
+          if (function_exists("drush_log")) {
```
Comment (RoSk0) [inline includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php]: You shouldn't use drush_log() directly in general functionality code, you should use watchdog() instead. drush_log() should be used only in Drush extensions.

File: includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php [hunk 2]
```diff
@@ -341,15 +347,22 @@ public function processOrphans($profileIds) {
...
+            drush_log("Deleted orphaned profile: " . $profile->label(), "status");
```
Comment (RoSk0) [inline includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php]: Don't forget to use t() and placeholders for dynamic parts, like `$profile->label()`.

File: includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php [hunk 3]
```diff
@@ -341,15 +347,22 @@ public function processOrphans($profileIds) {
...
+          }
           $profile->delete();
           break;
 
         case "unpublish":
           $profile->status->value(0);
           $profile->save();
-          drush_log("Unpublished orphaned profile: " . $profile->label(), "status");
+          if (function_exists("drush_log")) {
```
Comment (RoSk0) [inline includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php]: Same as above.

File: stanford_capx.blocks.inc [hunk 4]
```diff
@@ -147,3 +169,36 @@ function stanford_capx_data_browser_launch_block() {
 
   return $output;
 }
+
+/**
+ * Creates and displays the importer in use block.
+ * @return [type] [description]
```
Comment (RoSk0) [inline stanford_capx.blocks.inc]: This should be correctly filled in using https://www.drupal.org/coding-standards.

**Pull request merged**